site/src/components/Paywall/PopoverPaywall.stories.tsx

```diff
@@ -9,12 +9,19 @@ const meta: Meta<typeof PopoverPaywall> = {
 export default meta;
 type Story = StoryObj<typeof PopoverPaywall>;
 
-const Example: Story = {
+export const Enterprise: Story = {
   args: {
     message: "Black Lotus",
     description:
       "Adds 3 mana of any single color of your choice to your mana pool, then is discarded. Tapping this artifact can be played as an interrupt.",
   },
 };
 
-export { Example as PopoverPaywall };
+export const Premium: Story = {
+  args: {
+    message: "Black Lotus",
+    description:
+      "Adds 3 mana of any single color of your choice to your mana pool, then is discarded. Tapping this artifact can be played as an interrupt.",
```

Comment:
LOL, someone is playing too much Magic 😆

Reply:
Choosing the MVP of cards